dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -73,6 +73,7 @@ h1 {
 }
 
 h2 {
+  margin: 25px 0 10px 0;
```

Comment:
this seems like it has the potential to have very broad implications. Where else do we use `h2`s?